### PR TITLE
Moved CWL and WDL to constants

### PR DESCRIPTION
**Pull request by fhembroff**
**Description**
Whilst doing this [PR](https://github.com/dockstore/dockstore-cli/pull/231) I noticed that the strings `"WDL"` and `"CWL"` show up a lot in the CLI. So, this PR moves `"WDL"` and `"CWL"` to constants.

The constants that I am using (which I got from `io.dockstore.common.DescriptorLanguage`), have `WDL` and `CWL` capitalized. This not an issue, as our code permits any case to be used when dealing with the commands `CWL` and `WDL`. Also, if this was an issue, I imagine it would be picked up by our tests.

**Review Instructions**
Ensure that all tests run correctly.

**Issue**
I stated that I would do this in this [comment](https://github.com/dockstore/dockstore-cli/pull/231#discussion_r1134237050).

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [X] Check that you pass the basic style checks and unit tests by running `./mvnw clean install` 
- [X] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.

### CODE REVIEW

**Comment by fhembroff** [comment]
Sigh, it looks like sonar cloud is having some issues. The issue seems to be present in a few branches:

https://app.circleci.com/pipelines/github/dockstore/dockstore-cli/1488/workflows/c0a8fef3-2094-4722-8e28-1e1c838a72ec/jobs/6998
https://app.circleci.com/pipelines/github/dockstore/dockstore-cli/1490/workflows/bd2bb5d0-2065-4ec7-8493-71f9f8b6934b/jobs/7017

**Comment by fhembroff** [comment]
Looks like there is an outage for sonarcloud

https://sonarcloud.statuspage.io/

**Review by coverbeck — COMMENTED** [review]
>The constants that I am using (which I got from io.dockstore.common.DescriptorLanguage), have WDL and CWL capitalized. This not an issue, as our code permits any case to be used when dealing with the commands CWL and WDL. Also, if this was an issue, I imagine it would be picked up by our tests.

If there isn't already, could you please add a test or two where you pass a lower case value, e.g., `CWL.toString().toLowerCase()` just to confirm your statement and ensure it doesn't get broken in the future.

File: dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java [hunk 1]
```diff
@@ -371,28 +372,28 @@ void testTestJson() {
 
         // Update tag with test parameters
         Client.main(new String[] { CONFIG, ResourceHelpers.resourceFilePath("config_file.txt"), TOOL, TEST_PARAMETER, ENTRY,
-            "quay.io/dockstoretestuser/test_input_json", VERSION, "master", "--descriptor-type", "cwl", "--add", "test.cwl.json",
+            "quay.io/dockstoretestuser/test_input_json", VERSION, "master", "--descriptor-type", CWL.toString(), "--add", "test.cwl.json",
```
Comment (denis-yuen) [inline dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java:375]: Some of these `toString()` may be redundant. 
Couldn't find a better link, but https://youtrack.jetbrains.com/issue/IDEA-265548 gives the general idea.

**Review by denis-yuen — COMMENTED** [review]
Ok in general, think a bunch of `toString` is unnecessary. 
Can try to use the IDE to detect since it should detect any semantic changes

File: dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java (hunk 1 above)
Comment (fhembroff) [reply dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java:374]: I removed the redundant `toString()`s. 
[97c4b23](https://github.com/dockstore/dockstore-cli/pull/232/commits/97c4b23d0ac7924860fb328999bea8e84135fbbe) 
[6c059bd](https://github.com/dockstore/dockstore-cli/pull/232/commits/6c059bd3cb7cce326248eb7b49e4f953b74c86f4)

In the case of the selected code, it appears that the `.toString()` is required, as if I remove it, I get this error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:testCompile (default-testCompile) on project dockstore-cli-integration-testing: Compilation failure
[ERROR] /home/fhembroff/Documents/work/extract-cwl-wdl-to-constants/dockstore-cli/dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java:[374,98] incompatible types: io.dockstore.common.DescriptorLanguage cannot be converted to java.lang.String
[ERROR] 

```

**Comment by fhembroff** [comment]
> If there isn't already, could you please add a test or two where you pass a lower case value, e.g., `CWL.toString().toLowerCase()` just to confirm your statement and ensure it doesn't get broken in the future.

I have added three tests. [da78e5d](https://github.com/dockstore/dockstore-cli/pull/232/commits/da78e5d050c962f7ea419948bf1778822b60922d)

For `testTestJsonWithLowerCaseWDLandCWL` I duplicated `testTestJson`. Do you think it would be better to leave it how I did it, or should I create another method, where the values for CWL and WDL are variables, and I just make two calls to it (one with lower case WDL and CWL and one with upper case WDL and CWL)?

**Review by denis-yuen — APPROVED** [review]

**Review by coverbeck — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]

**Pull request merged**